### ISSUE TITLE
Is it possible to parametrize the frequency period in reporting

### ISSUE THREAD

**Issue by cimi** [bestiejs/benchmark.js#170]
I would like to be able to change the period for benchmark's reporting from one second to one minute (i.e. instead of having ops/sec I would like to have ops/min). I've just started playing with the library today, so apologies if this is a newbie question.

The reason is that in my application I process large datasets and I want to measure different implementations against each other. Processing those datasets takes significantly more than one second. Benchmarking against smaller datasets is possible, however, as far as I understand, it would not account for things like memory allocation / garbage collection which do have impact on performance in the real world.

I have gotten it to run with large datasets, however the output is not really helpful:

```
PhantomJS 2.1.1 (Mac OS X 0.0.0)  Processing data: iterative conversion and augmentation at 0 ops/sec
```

tl;dr How can I get it to say `32 ops/min`?

Thanks for the library by the way, it seems really useful!

**Comment by jdalton** [comment]
Hi @cimi,

You can explore the [stats](https://benchmarkjs.com/docs#prototype_stats) and [times](https://benchmarkjs.com/docs#prototype_times) properties on the benchmark object to customize the report to taste.
